### ISSUE TITLE
Metadata not getting generated in XML

### ISSUE THREAD

**Issue by kohsah** [cirsfid-unibo/lime#21]
The metadata block does not generated  in the XML ...tested both bill and doc types : 

```
<akomaNtoso xmlns="http://docs.oasis-open.org/legaldocml/ns/akn/3.0/CSD13" xmlns:html="http://www.w3.org/1999/xhtml">
    <bill>
        <preface>
            <p>Council
                <eol/>
                <eol/>Rome, 30 November – 
                <docDate date="2015-12-04">4 December 2015</docDate>
            </p>
        </preface>
        <body>
            <paragraph eId="para_1">
                <content eId="para_1__content">
                    <p>Item 2 - Election of three Vice-Chairpersons, and Designation of the Chairperson and Members of the Drafting CommitteeThe Council elected three Vice-Chairpersons for its Session: Mr Crisantos Obama Ondo (Equatorial Guinea), Mr Yousef Jhail (Kuwait) and Ms Gerda Verburg (Netherlands). </p>
                </content>
            </paragraph>
            <paragraph eId="para_2">
                <content eId="para_2__content">
                    <p>The Council elected Mr Matthew Worrell (Australia) as Chairperson of the Drafting Committee with the following membership: Afghanistan, Australia, Brazil, Canada, Ecuador, Iceland, Iraq, Madagascar, Malaysia, Republic of Korea, Russian Federation, San Marino, Spain, Thailand and Zimbabwe.</p>
                </content>
            </paragraph>
        </body>
    </bill>
</akomaNtoso>
```

See the attached screenshot, the metadata has been set for the document. Also the bill doc type is stock, I have not changed anything : 

![screenshot from 2016-03-04 23 28 47](https://cloud.githubusercontent.com/assets/5685392/13535564/ed6a7486-e260-11e5-8aca-d9fe854cd715.png)

**Comment by kohsah** [comment]
The problem seems to have started after commit babc60615a6858c84d31765a4a3408c20dbe491c .

When I reset HEAD to that commit, the metadata gets generated correctly.

**Comment by obujor** [comment]
I can't reproduce this bug, synchronize your repo and try again, it could be fixed with #19.

**Comment by kohsah** [comment]
OK Seems to have been fixed with Issue #19 , syncing with the updates, cannot simulate it either.
